.github/workflows/gardening.yml

```diff
@@ -0,0 +1,34 @@
+name: Repo gardening
+
+on:
+  # We need to listen to all these events to catch all scenarios
+  pull_request:
+    types: ['opened', 'synchronize', 'edited', 'closed', 'labeled']
+
+jobs:
+  repo-gardening:
+    name: 'Assign issues, Clean up labels, and notify Design when necessary'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+
+     - name: Setup Node
+       uses: actions/setup-node@v2
+       with:
+         node-version: 12
+
+     - name: Wait for prior instances of the workflow to finish
+       uses: softprops/turnstyle@v1
+       env:
+         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+     - name: 'Run gardening action'
+       uses: automattic/action-repo-gardening@master
```

Comment:
Once we're happy this works, we should tag a version in the action repo and update this. We ran into trouble with the review-required repo that using master elsewhere kinda tied our hands on if we could introduce any major changes (e.g. new config value needed)

Reply:
Yeah, that sound good to me. 👍

Reply:
Here is the update: #116